Source/OxyPlot/Series/FinancialSeries/CandleStickAndVolumeSeries.cs

```diff
@@ -517,8 +555,122 @@ protected internal override void UpdateData()
         /// </summary>
         protected internal override void EnsureAxes()
         {
-            base.EnsureAxes();
+            // find volume axis
             this.VolumeAxis = (LinearAxis)this.PlotModel.Axes.FirstOrDefault(a => a.Key == "Volume");
+
+            // assign the bar axis if not found
+            var barAxis = this.PlotModel.Axes.FirstOrDefault(a => a.Key == "Bars");
+            if (barAxis == null)
+            {
+                // determine a bar axis fallback
+                var defAxis = this.PlotModel.Axes.FirstOrDefault(a => a.IsVertical() && a.Key != "Volume");
+               
+                // would be nice to avoid having to change the key & override base method
+                defAxis.Key = "Bars";
```

Comment:
My preference would be not to do this, but to be able to override the axis search.  If volume is not to be displayed, would like to have it default to the 1st YAxis.   CandleStickAndVolumeSeries can be used to just display candlesticks.  

Would you suggest making the keying mandatory for just bar display (as would be the default without this) or finding another way to assign default?    I wanted to override the base method where these are assigned but could not write to the YAxis property.